modules/assisted-installer-configuring-a-static-network.adoc

```diff
@@ -0,0 +1,30 @@
+// This is included in the following assemblies:
+//
+// assisted-installer-installing.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-a-static-network_{context}"]
+= Optional: Configuring a static network
+
+The {ai-full} supports IPv4/IPv6 dual stack networking and NMState-based static IP addressing.
```

Comment:
NMstate does not mean static. It still can use dhcp as part of the nmstate configuraiton. I would say more of nmstate advanced networking configuraiton, such as static ips, bonds, vlans, etc...